src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -496,7 +498,11 @@ class GridEditable<
           {/* TODO(leedongwei): This is ugly but I need to move it to work on
           resizing columns. It will be refactored in a upcoming PR */}
           <div style={{display: 'flex', flexDirection: 'row'}}>
-            {this.renderHeaderButton()}
+            <HeaderButton onClick={downloadAsCsv} data-test-id="grid-add-column">
+              <InlineSvg src="icon-download" />
+              {t('Download CSV')}
+            </HeaderButton>
+            <div style={{marginLeft: '16px'}}>{this.renderHeaderButton()}</div>
```

Comment:
There are a few marginLeft elements now, can we make a styled component for these? Something like

```typescript
const HeaderButtonContainer = styled('div')`
  margin-left: ${space(2)};
`;
```